pisek/jobs/parts/solution.py

```diff
@@ -343,7 +352,7 @@ def subtask_name(num: int) -> str:
             )
             text += "\n"
 
-        if len(self.subtask.predecessors) and any(
+        if len(self.subtask.all_predecessors) and any(
```

Comment:
If I'm not mistaken, this is a change in behavior, since `predecessors` is now `direct_predecessors`. Was that intended?

Reply:
Yup. I found it clearer to include all predecessors in double verbose mode.